optuna_dashboard/static/action.ts

```diff
@@ -33,8 +33,14 @@ export const actionCreator = () => {
   }
 
   const updateStudyDetail = (studyId: number) => {
-    getStudyDetailAPI(studyId)
+    getStudyDetailAPI(
+      studyId,
+      studyId in studyDetails ? studyDetails[studyId].trials.length : 0
```

Comment:
`RUNNING` and `WAITING` trials will be updated. So you need to pass the last completed trial number.